src/QMCDrivers/WFOpt/HamiltonianRef.cpp

```diff
@@ -75,7 +75,7 @@ int HamiltonianRef::addObservables(ParticleSet& P)
 void HamiltonianRef::evaluate(MCWalkerConfiguration& W, std::vector<RealType>& energyVector)
 {
   using WP = WalkerProperties::Indexes;
-  std::vector<Walker_t*>& walkers = W.WalkerList;
+  auto& walkers                   = W.WalkerList;
```

Comment:
strange space.

Reply:
It is part of a two line list with the = aligned in the same column.